tools/fileconv/api3/queries.go

```diff
@@ -15,20 +15,44 @@ type Explorer struct {
 	*parameters
 }
 
-// GetBasicReadObjects retrieves schemas that can be used by ListObjectMetadata.
+// ReadObjectsGet retrieves schemas that can be used by ListObjectMetadata.
 // objectEndpoints - optional map of Endpoint Path to ObjectName associated with it.
 // ignoreEndpoints - optional list of paths to ignore
 // displayNameOverride - optional map of ObjectName to DisplayName. This will override display name from OpenAPI doc.
-func (e Explorer) GetBasicReadObjects(
+func (e Explorer) ReadObjectsGet(
 	ignoreEndpoints []string,
 	objectEndpoints map[string]string,
 	displayNameOverride map[string]string,
 	check ObjectCheck,
-) ([]Schema, error) {
-	schemas := make([]Schema, 0)
+) (Schemas, error) {
+	return e.ReadObjects("GET", ignoreEndpoints, true, objectEndpoints, displayNameOverride, check)
+}
+
+// SearchObjectsPost is the same as ReadObjectsGet but retrieves schemas for endpoints that perform reading via POST.
+func (e Explorer) SearchObjectsPost(
+	fromEndpoints []string,
```

Comment:
If this is the same as `ReadObjectsGet.ignoreEndpoints`, can you call it the same? `fromEndpoints` isn't very clear on what it does